SConstruct

```diff
@@ -120,6 +120,9 @@ if ARGUMENTS.get('more') or ARGUMENTS.get('all'):
     AddBuilds("noio", allfeatures + noioconfigs + cflags)
     AddBuild("debug-size", ["-DMPACK_OPTIMIZE_FOR_SIZE=1"] + debugflags + allfeatures + allconfigs + cflags)
 
+# C11
+if ARGUMENTS.get('c11'):
+    AddBuild("c11", allfeatures + allconfigs + releaseflags + ["-std=c11"])
```

Comment:
This is fine. It currently compiles in C99 by default because I figured it would ensure better compatibility, but since we're adding C11 features, it probably makes more sense to make C11 the default. So I might switch to that later; in the meantime it's good like this.

Also note that the Travis build nodes don't support C11 (they only have GCC 4.6.3), and I just noticed that Travis didn't even build the pull request. Not sure why, but it's probably not important since it won't actually be testing _Generic anyway. We can just run the tests manually before merging.